app/services/privilege_checker_service.rb

```diff
@@ -22,6 +22,6 @@ def session_active?(session)
   end
 
   def server_ready?(session)
-    MiqServer.my_server(true).logon_status == :ready || session[:userrole] == "super_administrator"
+    MiqServer.my_server(true).logon_status == :ready || super_admin_user?
```

Comment:
Interesting...if you flip this logic, then it hits the database less frequently in super admin cases...it's funny how refactoring makes things like this pop up.